libr/core/cmd_print.c

```diff
@@ -489,6 +489,97 @@ R_API int r_core_process_input_pade(RCore *core, const char *input, char** hex,
 	return result;
 }
 
+static void task_manage_help(RCore *core) {
```

Comment:
as long as they are static, we use camelcase syntax, and will be good if all the functions that do similar thing start with the same words. for example:

* helpCmdTasks(RCore*)
* helpCmdForeach()
* helpCmdAt()